tests/integration/rkeMachineConfig_test.go

```diff
@@ -1,12 +1,18 @@
 package integration_test
 
 import (
+	"os"
+	"runtime"
+
 	"github.com/rancher/webhook/pkg/auth"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
 func (m *IntegrationSuite) TestRKEMachineConfig() {
+	if runtime.GOARCH == "arm64" && os.Getenv("CI") != "" {
+		m.T().Skip("Skipping the RKE Machine-Config test on arm64 in CI -- machine info not available")
+	}
```

Comment:
nit: Could add a comment to describe why we're skipping a test

Reply:
Sure